development/hurl/scenarios/setup.hurl

```diff
@@ -672,8 +672,42 @@ HTTP 201
 [Captures]
 ss1_auth_key_cert_hash: jsonpath "$.certificate_details.hash"
 
+# Get not available hardware token
```

Comment:
Is this the token softHSM generates by default? Maybe it could be deleted from entrypoint script instead?